ui/src/containers/groups.list/index.tsx

```diff
@@ -10,17 +10,26 @@ import { GroupsHeader } from "./header";
 type ContextType = { group: V1Beta1Group | null };
 export default function GroupList() {
   const { client } = useFrontier();
-  const [groups, setGroups] = useState([]);
+  const [groups, setGroups] = useState<V1Beta1Group[]>([]);
+  const [isGroupsLoading, setIsGroupsLoading] = useState(false);
+
   useEffect(() => {
     async function getGroups() {
-      const {
-        // @ts-ignore
-        data: { groups },
-      } = await client?.adminServiceListGroups();
-      setGroups(groups);
+      setIsGroupsLoading(true);
+      try {
+        const {
+          // @ts-ignore
+          data: { groups },
+        } = await client?.adminServiceListGroups();
+        setGroups(groups);
+      } catch (err) {
+        console.error(err);
```

Comment:
we are logging error instead lets use toast. Other place as well

Reply:
lets do this in other PR